### DISCUSSION TITLE
x-init after tailwind loaded.

### DISCUSSION THREAD

**Discussion by mrl22** [alpinejs/alpine#4358] [1. Help]
I was wondering if it was possible to get x-init to run after Tailwind has loaded.

I have been working on a AlpineJS component that allows a user to slide content panels. Example here: [https://splitter.devfwd.com/set-size](https://splitter.devfwd.com/set-size)

```
<x-splitter-h>
    <x-slot name="first" class="">

        Panel 1 Text

    </x-slot>
    <x-slot name="second" class="h-40">

        Panel 2 Text

    </x-slot>
</x-splitter-h>
```

I am trying to allow the user to set a onload/startup tailwind class width/height of a panel, but x-init is reading the panel width/height as 0 because tailwind has not yet loaded.

The project can be found here: [https://github.com/mrl22/alpinejs-splitter/blob/main/resources/views/components/splitter-h.blade.php](https://github.com/mrl22/alpinejs-splitter/blob/main/resources/views/components/splitter-h.blade.php)

Any help is appreciated.

**Comment by ekwoka** [comment]
You should load your CSS....before the page renders...

**Reply by mrl22** [reply]
Thanks for your feedback @ekwoka , that would be render blocking?

**Reply by ekwoka** [reply]
Yes, normal link tags for stylesheets are render blocking
